plugins/com.google.cloud.tools.eclipse.util/templates/appengine/MockHttpServletResponse.java.ftl

```diff
@@ -147,31 +147,31 @@ public class MockHttpServletResponse implements HttpServletResponse {
   }
```

Comment:
Above this methods, `sendError(int arg0)` --> `sendError(int statusCode`).

Reply:
This is still not fixed. There's a method with `arg0` in the file.

Reply:
done